src/api/app/views/webui/project/_form.html.haml

```diff
@@ -7,9 +7,9 @@
       .input-group-prepend
         %span.input-group-text
           #{namespace}:
-      = form.text_field(:name, required: true, class: 'form-control')
+      = form.text_field(:name, required: true, class: 'form-control', autofocus: true)
   - else
-    = form.label(:name, 'Project Name:')
+    = form.label(:name, 'Project Name:', autofocus: true)
```

Comment:
`autofocus: true` needs to be on the text field

Reply:
So true, changing